pkg/test/framework/components/istio/config.go

```diff
@@ -130,6 +131,10 @@ type Config struct {
 
 	// Expose istiod through ingress, for example for multicluster or VM setups
 	ExposeIstiod bool
+
+	// 	If false,  all remote clusters will be the new istiod-less remotes.
```

Comment:
nit: comment seems to start with extra spaces

Reply:
fixed